demo/index.html

```diff
@@ -287,7 +287,6 @@ <h1 class="text-3xl font-bold">
           description: 'Late af task',
           size: 10,
           x: 12.069770990416055,
-          y: 12.069770990416057,
```

Comment:
I kinda like your desire to remove the `y` value :smile: 

But, if I were you I would keep only one `y` value in the demo to demonstrate that it is possible to use custom `y` value.